js/views/adScreen.js

```diff
@@ -187,6 +202,15 @@ var AdScreen = React.createClass({
           {playbackControlItems}
         </div>
 
+        {showUnmute ? <button className="oo-ad-screen oo-unmute"
+          type="button"
+          tabindex="0"
```

Comment:
Sorry, my bad. I think this actually needs to be `tabIndex` with capital i for React

Reply:
Done